disaster_recovery.rst

```diff
@@ -50,31 +50,63 @@ If so, this information will make your work a lot easier.
-* The domain controller is needed so the users are able to sign in to the ERP. However, since 002 and 003's main purposes is load balancing, recovering **dcsrv001** will be our main objective.
-* The fileserver **stgsrv001** is used to store copies of the order receipts produced in the ERP.
+* The application server **app-srv-001** is running our ERP, which is needed to be able to place orders from customers.
+* The SQL Server **sql-srv-001** contains the data from the ERP which means that it's a prerequisite for the application server.
+* The domain controller is needed so the users are able to sign in to the ERP. However, since 002 and 003's main purposes is load balancing, recovering **dc-srv-001** will be our main objective.
+* The fileserver **stg-srv-001** is used to store copies of the order receipts produced in the ERP.
 
 What we've concluded from this activity is that we need to recover four components to be able to use the ERP.
 Note however, that in reality, the IT environment's usually alot more complex then the one used in this example.
 
 The identified objects then need to be ranked to determine in what order they need to be recovered to minimize downtime.
 In this example, this would most likely be as follows:
 
-1. dcsrv001
-2. sqlsrv001
-3. stgsrv001
-4. appsrv001
+* dc-srv-001
+* sql-srv-001
+* stg-srv-001
+* app-srv-001
 
-.. TODO:: text - How to prioritize recovery components, shared resources, bussiness needs.
+Prioritizing Recovery Components
+--------------------------------
+In the example above, the number of assets to prioritize is low, which might suggest that there is no need for making a priority list.
+However, one thing that you'll learn by doing a couple of disaster recovery excercises is that no matter how small the scope, stakeholders will always try to direct your effors to the assets mosts relevant to them.
+For example, the CEO might want you to prioritize the ERP System over the Domain Controller, which might very well be correct, but as the list of assets grow longer the number of stakeholders wanting to influence your decisions will aswell.
+
+As you might have guessed, it may be a good idea to actually prioritize your asset list, if not out of necessity then atleast to circumvent any issues that might occur because of differing opinions on what needs to be prioritized.
+A great way to start out is to create a spreadsheet consisting of the following columns:
+
+* Asset Identification
+    A name, a FQDN or an IP.
+    Whatever helpes you identifying the asset.
+ 
+* Business Priority
+    Non-Essential, Essential, Critical.
+    This should be decided buy either the board or the senior management team.
+
+* Tiebreaker/Sequential numbering
+    A sequential numbering which will break ties in case multiple assets have the same priority.
+    Should also be decided by the board or the senior management team.
```

Comment:
I think I might have to clarify a bit in the text. Think of it like this:

If ABC and XYZ both have the same priority, which one would you like us to prioritize if they both are tied for our attention?

Makes sense?